android/app/src/main/java/com/expensify/chat/MainApplication.java

```diff
@@ -49,6 +51,17 @@ public void onCreate() {
       if (BuildConfig.DEBUG) {
           FirebaseCrashlytics.getInstance().setCrashlyticsCollectionEnabled(false);
       }
+
+      // Increase SQLite DB write size
+      try {
+        Field field = CursorWindow.class.getDeclaredField("sCursorWindowSize");
+        field.setAccessible(true);
+        field.set(null, 100 * 1024 * 1024); //the 100MB is the new size
+      } catch (Exception e) {
+        if (BuildConfig.DEBUG) {
```

Comment:
Good idea. I adapted this line from an issue comment. Do we have a way to log things on the native Android side? I'm up for this but not really sure how it works or if we are set up for that yet. Should we just let it throw?

Reply:
It's way too dangerous to just let it throw, catching and printing the stack trace is the correct action here as it will clearly be visible to anyone monitoring native logs via Android Studio or ADB.

> Do we have a way to log things on the native Android side?

Not 100% sure that this is what you're asking -- but we could log a more detailed message with Log.e('MainApplication', 'Failed to increase SQLite write size'). The stack trace alone should be enough to locate this should be enough thought IMO.

Reply:
So definitely remove the condition, continue to print the stack trace, and maybe add a Log for additional help.

Reply:
To clarify, you'd like me to leave things as they are..  but add a `Log.e('MainApplication', 'Failed to increase SQLite write size')` to the code? Is the logcat something we will see somewhere? e.g. if this happens to our users will that make it easier to identify?

Reply:
I think this is probably OK for now maybe it will never throw. But lmk if you feel different.

Reply:
Sorry, I should have been clearer. The only change I think you should make is to remove the condition which prevents the error stack trace from being printed. It probably will never throw, but there is no reason to hide the stack trace if it does.

As for the Log, I thought you were asking if it was possible -- I think we both agree that it is not necessary or useful.